base/src/main/java/org/kontinuity/catapult/base/identity/IdentityVisitor.java

```diff
@@ -0,0 +1,15 @@
+package org.kontinuity.catapult.base.identity;
+
+/**
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+public interface IdentityVisitor {
```

Comment:
Docs for what implementations should do here and what users should expect when they call upon the business methods.